client/my-sites/themes/themes-banner/index.jsx

```diff
@@ -0,0 +1,53 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import cssSafeUrl from 'lib/css-safe-url';
+
+export class ThemesBanner extends Component {
```

Comment:
Let's make it extend `PureComponent` since its appearance and behavior are fully determined by its props. (This allows React to only re-render when props change, which is good for performance, and makes sense here.)